src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -96,59 +96,69 @@ You can use NerdGraph to return a wide range of New Relic data but we've created
   </thead>
 
   <tbody>
-    <tr>
+   <tr>
       <td>
-        Alerts
+**NerdGraph basics**
       </td>
 
       <td>
-        [See all alert-related tutorials](/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-nrql-condition-alerts/)
+There are some of the most commonly used NerdGraph features:
+* [Query your data using NRQL](/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial) (our query language)
+* [Get data about monitored entities](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial)
+* [Create dashboards](/docs/apis/nerdgraph/examples/create-widgets-dashboards-api/)
```

Comment:
@zuluecho9, I'm not sure this basics category is helpful since you've pulled out the dashboards and entities material from their respective categories. Would you consider pushing the dashboards and entities items back into their respective categories and just having a NRQL category? NRQL doesn't see super basic to me, anyway.

Reply:
thanks, i'll do some more reconfiguring